src/main/java/com/atomist/rug/cli/command/AbstractParameterizedCommand.java

```diff
@@ -53,7 +53,7 @@ private ParameterValues collectParameters(ParameterizedRug rug, ParameterValues
                 String defaultValue = (pv != null ? pv.getValue().toString()
                         : parameter.getDefaultValue());
 
-                log.info("  " + WordUtils.wrap(parameter.getDescription(), Constants.WRAP_LENGTH,
+                log.info("  " + WordUtils.wrap(capitalizeFirstLetter(parameter.getDescription()), Constants.WRAP_LENGTH,
```

Comment:
Good call. I was looking at `WordUtils.capitalize()` and forgot about `StringUtils.capitalize()`